Seeing this error when building the compiler-rt:

```
/Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.1-branch/llvm/projects/compiler-rt/lib/builtins/os_version_check.c:231:47: error: use of undeclared identifier 'initializeAvailabilityCheck'
23:53:07   dispatch_once_f(&DispatchOnceCounter, NULL, initializeAvailabilityCheck);
23:53:07                                               ^
23:53:07 /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-swift-5.1-branch/llvm/projects/compiler-rt/lib/builtins/os_version_check.c:251:47: error: use of undeclared identifier 'initializeAvailabilityCheck'
23:53:07   dispatch_once_f(&DispatchOnceCounter, NULL, initializeAvailabilityCheck);
23:53:07                                               ^
23:53:07 2 errors generated.
```

--- a/lib/builtins/os_version_check.c
+++ b/lib/builtins/os_version_check.c
@@ -88,7 +88,7 @@ typedef Boolean (*CFStringGetCStringFuncTy)(CFStringRef, char *, CFIndex,
 typedef void (*CFReleaseFuncTy)(CFTypeRef);
 
 /* Find and parse the SystemVersion.plist file. */
-static void parseSystemVersionPList(void *Unused) {
+static void initializeAvailabilityCheck(void *Unused) {
   (void)Unused;
 
   // Use the new API if it's is available. Still load the PLIST to ensure that the

